packages/vsce/src/commands/inspectTreeResourceCommand.ts

```diff
@@ -13,6 +13,9 @@ import { ExtensionContext, TreeView, commands, window } from "vscode";
 import { IResource } from "../doc";
 import { CICSResourceContainerNode } from "../trees";
 import { inspectResourceByNode } from "./inspectResourceCommandUtils";
+import { ResourceTypes } from "@zowe/cics-for-zowe-explorer-api/lib/resources/ResourceTypes";
+import { EventSourceTypes } from "@zowe/cics-for-zowe-explorer-api/lib/enums/EventSourceTypes";
+import { IResourceInspectEvent } from "@zowe/cics-for-zowe-explorer-api/lib/interfaces/IResourceInspectEvent";
```

Comment:
You should, if things are exported in index files correctly, be able to condense all this
```suggestion
import { ResourceTypes, EventSourceTypes, IResourceInspectEvent } from "@zowe/cics-for-zowe-explorer-api";
```